### PR TITLE
Enable v1.13 code slush.

### PR DESCRIPTION
**Pull request by amwat**
Split k/k tide queries in preparation for code slush.

/hold till 11/09
/cc @cjwagner

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -400,6 +400,30 @@ tide:
...
+    - do-not-merge
+    - do-not-merge/blocked-paths
+    - do-not-merge/cherry-pick-not-approved
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/release-note-label-needed
+    - do-not-merge/work-in-progress
+    - needs-kind
+    - needs-rebase
+    - needs-sig
```
Comment (cjwagner) [inline prow/config.yaml]: This also needs to include `needs-priority`

Comment (amwat) [reply prow/config.yaml]: Done.

**Pull request merged**